app/assets/javascripts/components/PipelineSampleReport.jsx

```diff
@@ -304,7 +304,10 @@ class PipelineSampleReport extends React.Component {
                         </td>
 
                         <td>
-                          { (this.view_level==='species' && (taxon.nt_ele && taxon.nt_ele.hasOwnProperty('count')) ) ? numberWithCommas(taxon.nt_ele.count) : '' }
+                          { (this.view_level==='species' && (taxon.nt_ele && taxon.nt_ele.hasOwnProperty('count')) ) ? 
+                            <span className="link">
+                              <a href={`/pipeline_outputs/${this.pipeline_output_id}/${taxon.nt_ele.tax_id}/fasta`}>{ numberWithCommas(taxon.nt_ele.count) }</a>
+                            </span> : '' }
```

Comment:
I hope I don't lose this detail in my refactor of this page :)   Watch out when you review my diff tomorrow.